### PR TITLE
Support portal subdomain and context path deployment

### PR DESCRIPTION
**Pull request by DeaDurro**

### CODE REVIEW

**Comment by DeaDurro** [comment]
is it a problem if the backend path is /api when then "api": {
    "baseUrl": "/api" in the deploy-config.json.default?

File: feasibility-portal/proxy/docker-compose.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ services:
     volumes:
       - ${PROXY_CERTIFICATE_PATH:-../auth/cert.pem}:/etc/nginx/conf.d/cert.pem
       - ${PROXY_CERTIFICATE_KEY_PATH:-../auth/key.pem}:/etc/nginx/conf.d/key.pem
-      - ${PROXY_NGINX_CONFIG_PATH:-./nginx.conf}:/etc/nginx/nginx.conf:ro
-      - ./conf.d:/etc/nginx/templates
+      - ${PROXY_NGINX_CONFIG_PATH:-./context-paths.nginx.conf}:/etc/nginx/nginx.conf:ro
```
Comment (EmteZogaf) [inline feasibility-portal/proxy/docker-compose.yml:14]: Please remove whitespaces at the end of line.

File: feasibility-portal/README.md [hunk 2]
```diff
@@ -66,6 +66,26 @@ The portal is configured by default to start the following services:
...
+- Change the values for the variables `FEASIBILITY_BACKEND_API_BASE_URL` in [backend/.env](./backend/.env) and `FEASIBILITY_BACKEND_ALLOWED_ORIGINS` in [backend /.env](./backend/.env)
+          to the base url of your feasibility portal backend. In the [backend/.env](./backend/.env) change the values for the variable `FEASIBILITY_BACKEND_KEYCLOAK_BASE_URL_ISSUER`	to the base url of your feasibility portal keycloak.
+      
+  
+
+
+
+
+
+
```
Comment (EmteZogaf) [inline feasibility-portal/README.md:88]: Reduce empty lines to 1 or maybe 2.

**Review by EmteZogaf — CHANGES_REQUESTED** [review]
The 3 commits need actual [commit messages](https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53) and can probably be squashed into 1 commit.

**Review by EmteZogaf — APPROVED** [review]

**Pull request merged**